tendrl/monitoring_integration/alert/handlers/cluster/gluster/brick_utilization_handler.py

```diff
@@ -42,8 +42,8 @@ def format_alert(self, alert_json):
             if alert['severity'] == "WARNING":
                 alert['tags']['message'] = (
                     "Brick utilization of %s in node %s in "
```

Comment:
I feel its better to show the brick name as single entry in the form `{node-fqdn}:{brick path}`.
The message in current form looks too big.